.github/workflows/main.yml

```diff
@@ -88,7 +109,7 @@ jobs:
 
       - name: Build
         run: |
-          cd build && make
+          cd build && VERBOSE=1 make -j4
```

Comment:
I poked through the logs; looks like ccache is being applied to all the C++ files that are being compiled by CMake, and also during the R package build.  So I guess we can remove the verbose now?  Not sure if you had anything else you wanted to check.

Reply:
Perfect.  It was mostly there for us to check.  We know where to turn it on.

And I will assign myself four brownie points for smuggling `-j 4` in.

Reply:
Done in 95cf33f6a7